.github/workflows/release-draft.yml

```
@@ -0,0 +1,150 @@
+      - name: Run tests
+        run: |
+          poetry run make test
+          poetry run make lint
+          poetry run make type-check
+
+      - name: Build package
+        run: poetry build
+
+      - name: Extract version
+        id: version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)